db.go

```diff
@@ -240,22 +234,16 @@ func (self *DB) resp(_ string, params []interface{}, res interface{}) (interface
 		}
 
 		return arr[0], nil
-
 	}
 
 	return res, nil
-
 }
 
 func isSlice(possibleSlice interface{}) bool {
 	slice := false
-
-	switch v := possibleSlice.(type) {
-	default:
-		res := fmt.Sprintf("%s", v)
-		if res == "[]" || res == "&[]" || res == "*[]" {
-			slice = true
-		}
+	res := fmt.Sprintf("%s", possibleSlice)
```

Comment:
The original code was a `switch` over a `.(type)`, whilst yours is on the value.